ui/src/main/webapp/src/app/executions/executions-layout.component.ts

```diff
@@ -117,6 +117,16 @@ export class ExecutionsLayoutComponent extends ProjectLayoutComponent implements
                 this.execution,
                 'migration-issues'
             ),
+            new ContextMenuItem(
+                'Static Reports',
+                'fa-external-link',
+                true,
+                WindupExecutionService.formatStaticReportUrl(this.execution),
+                null,
+                null,
+                '_blank',
+                true
+            )
```

Comment:
I looked at wikipedia and they use this SVG image for new window: 
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 11 15" width="15" height="11">
    <g id="magnify-clip" fill="#fff" stroke="#000">
        <path id="bigbox" d="M1.509 1.865h10.99v7.919h-10.99z"/>
        <path id="smallbox" d="M-1.499 6.868h5.943v4.904h-5.943z"/>
    </g>
</svg>
```
![new window](https://en.wikipedia.org/w/resources/src/mediawiki.skinning/images/magnify-clip-ltr.png) 

and this one for external link:
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12">
  <path fill="#fff" stroke="#06c" d="M1.5 4.518h5.982V10.5H1.5z"/> 
  <path d="M5.765 1H11v5.39L9.427 7.937l-1.31-1.31L5.393 9.35l-2.69-2.688 2.81-2.808L4.2 2.544z" fill="#06f"/>
  <path d="M9.995 2.004l.022 4.885L8.2 5.07 5.32 7.95 4.09 6.723l2.882-2.88-1.85-1.852z" fill="#fff"/>
</svg>
```
![external link](https://en.wikipedia.org/w/skins/Vector/images/external-link-ltr-icon.png?325de)